utils/e2e-tests/ts/lib/node.ts

```diff
@@ -6,6 +6,7 @@ import { AddCleanup } from "./cleanup";
 
 export type RunNodeParams = {
   args: string[];
+  env?: NodeJS.ProcessEnv | undefined;
```

Comment:
```suggestion
  env?: NodeJS.ProcessEnv;
```

`env?: ...` implies `undefined`